rosbag_snapshot/src/snapshot.cpp

```diff
@@ -120,6 +122,8 @@ bool parseVariablesMap(SnapshotterOptions& opts, po::variables_map const& vm)
   opts.default_duration_limit_ = ros::Duration(vm["duration"].as<double>());
   opts.default_count_limit_ =  vm["count"].as<int32_t>();
   opts.all_topics_ = vm.count("all");
+  if (vm.count("compression"))
```

Comment:
If we use the default value, then we don't need this check any longer.

Reply:
It worked, cool!